packages/calling-component-bindings/src/participantListSelector.ts

```diff
@@ -94,6 +98,8 @@ export type ParticipantListSelector = (
   participants: CallParticipantListParticipant[];
   myUserId: string;
   totalParticipantCount?: number;
+  /* @conditional-compile-remove(rooms) */
+  myRole?: Role;
```

Comment:
Do we need this to show the local user's role in the participant list?

Reply:
was to simplify the internal logic of the participant list. we can also map against the users Id.